core/src/main/scala/org/apache/pekko/persistence/cassandra/query/EventsByPersistenceIdStage.scala

```diff
@@ -16,16 +16,16 @@ package org.apache.pekko.persistence.cassandra.query
 import com.datastax.oss.driver.api.core.CqlSession
 import com.datastax.oss.driver.api.core.cql._
 import org.apache.pekko
-import pekko.Done
-import pekko.annotation.InternalApi
-import pekko.persistence.cassandra.PluginSettings
-import pekko.stream.stage._
-import pekko.stream.{ Attributes, Outlet, SourceShape }
+import org.apache.pekko.Done
```

Comment:
Can we keep the `pekko` prefix here rather than using `org.apache.pekko`. While there is an argument to use the FQCN the community needs to decide on this and it will likely be done as part of a scalafix integration.

Reply:
will revert, organize import automatically broke that